### PR TITLE
Improve the data used for a v6 dev network so that more algorithms can be run on it

### PR DESCRIPTION
**Pull request by bartvanb**

### CODE REVIEW

**Review by frankcorneliusmartin — APPROVED** [review]
I am also OK with how it is now. But please check that the `.csv` is included when you package it (I dont think it is the case now)

File: vantage6/vantage6/cli/dev/create.py [hunk 1]
```diff
@@ -17,49 +19,45 @@
...
-        writer = csv.writer(f)
-
-        # write the header
-        writer.writerow(header)
-
-        # write the data
-        for row in data:
-            writer.writerow(row)
+    data_files = []
+    current_dir = Path(__file__).parent
```
[the remark this answers is not in the document]
Comment (bartvanb) [reply vantage6/vantage6/cli/dev/create.py:38]: I added `"dev/data/olympic_athletes_2016.csv",` to the package_data (which already contained several other files)

[the remark this answers is not in the document]
Comment (frankcorneliusmartin) [reply vantage6/vantage6/cli/dev/create.py:39]: Did you also use the `pkgutil.get_data`?

[the remark this answers is not in the document]
Comment (bartvanb) [reply vantage6/vantage6/cli/dev/create.py:39]: That's deprecated: https://setuptools.pypa.io/en/latest/pkg_resources.html#package-discovery-and-resource-access-using-pkg-resources, see also: https://stackoverflow.com/a/20885799/5398197. I'll adjust to using the recommended practice there

**Pull request merged**